envoy/USAGE.md

```diff
@@ -0,0 +1,25 @@
+## Purpose
+
+This directory contains a Dockerfile that builds Envoy with the [OQS BoringSSL master-with-bazel branch](https://github.com/Post-Quantum-Mesh/boringssl) modified to build the liboqs library and use the most updated BoringSSL source code.
+
+## Pre-Built Docker Image
+
+A pre-built [Docker image](https://hub.docker.com/layers/drouhana/envoy-oqs/envoy/images/sha256-e779ccfd8707e31fbf3f47f1f2ac99cb52ea56f6e923a87fbb12b7fa1dbca114?context=repo) has been provided for streamlined use in envoy implementations.
+
+## Local Docker Build
+
+Install [Docker](https://docs.docker.com/get-docker/) and run the following commands:
+
+    docker build -t envoy-oqs .
+
+## Using the Image
```

Comment:
Only this section should go into USAGE.md. This part would be published on hub.docker.com and should explain the usage of the image. Therefore a bit more explanation of (or a link to other documentation for) "standard envoy images" would be great.

Reply:
> images can be used identically to the standard envoy images

Can you please add a link to such documentation? dockerhub doesn't contain info at https://hub.docker.com/r/envoyproxy/envoy

Reply:
@drouhana Did you see the question above?

Reply:
@drouhana : OK to add a link to https://www.envoyproxy.io/docs/envoy/latest/start/docker (as documentation for "standard envoy image" use)? If you don't object I'll do that manually in the docker hub documentation (unless you already add it in this PR/USAGE.md).

Reply:
@baentsch Perfectly fine! Thank you very much for doing that

Reply:
So done: https://hub.docker.com/repository/docker/openquantumsafe/envoy/general